addons/binding/org.openhab.binding.sensebox/README.md

```diff
@@ -0,0 +1,114 @@
+# senseBox Binding
+
+The senseBox binding integrates environment data from the [senseBox](https://sensebox.de/)
+Citizen Science Toolkit.
+
+## Prerequisites
+
+The API server uses Letsencrypt certificates. Therefore, one needs to either import the Letsencrypt
+root certificates into the local keystore (see the description in the
+[netatmo addon](http://docs.openhab.org/addons/bindings/netatmo/readme.html)).
+Another way would be to simply update the JDK to at least JDK 1.8.0_111
+
+## Supported Things
+
+This binding supports a generic "senseBox" API endpoint which is a representation of the physical box.
+
+## Discovery
+
+This binding provides no discovery. The desired senseBox must be configured manually or via a things file.
+
+## Binding configuration
+
+The binding has no configuration options itself, all configuration is done at 'Things' level.
+
+## Thing configuration
+
+The senseBox thing requires the box Id (which can be obtained on the map) and an interval in seconds
+for the API polling. The senseBox support team wrote in an email that polling even every five minutes
+is still o.k., therefore the minimum is hardcoded to be 300 seconds.
+
+```
+Thing sensebox:box:home [ senseBoxId = "foothesmurfingbar", refreshInterval = 600 ]
+```
+
+## Channels
+
+In the table is shown more detailed information about each Channel type.
+The binding introduces the following channels:
+
+| Channel ID | Channel Description | Supported item type | Advanced |
+| ------------- | ------------- |------------|----------|----------|
+| descriptors#name | Name of the box | String | False |
+| descriptors#location | Location of the box | Point | False |
+| descriptors#exposure | Whether the box is mounted inside or outside | String | False |
+| descriptors#apiUrl | URL to the data API | String | True |
+| descriptors#imageUrl | URL to the image of the box on the map, for display purposes | String | True |
+| descriptors#mapUrl | URL to the box on the map, for display purposes | String | True |
+| measurements#uvIntensity | Intensity of Ultraviolet radiation | Number | False |
+| measurements#luminance | Illuminance | Number | False |
+| measurements#pressure | Air pressure | Number | False |
+| measurements#humidity | Humidity | Number | False |
+| measurements#temperature | Temperature | Number | False |
+| lastReported#uvIntensityLastReported | The timestamp when uv radiation intensity was last reported | DateTime | True |
+| lastReported#luminanceLastReported | The timestamp when illuminance was last reported | DateTime | True |
+| lastReported#pressureLastReported | The timestamp when pressure was last reported | DateTime | True |
+| lastReported#humidityLastReported | The timestamp when humidity was last reported | DateTime | True |
+| lastReported#temperatureLastReported | The timestamp when temperature was last reported | DateTime | True |
+
+The channels starting with "descriptors" are defined on the API server.
+
+## Channel configuration
+
+...
```

Comment:
Well, your channels do not have any configuration parameters, so there is no need for this section anyhow :-)